Contents/Code/__init__.py

```diff
@@ -338,7 +358,29 @@ def normalize_name(self, input_name):
             input_name
         )
         return normalizedName
-
+    
+    def name_to_initials(self, input_name):
+        # Shorten input_name by splitting on whitespaces
+        # Only the surname stays as whole, the rest gets truncated
+        # and merged with dots.
+        # Example: 'Arthur Conan Doyle' -> 'A.C.Doyle'
+        nameparts = input_name.split()
+        newName = ""
```

Comment:
Stay consistent with your variable casing. Python prefers snake case (snake_case, camelCase). This also mean `nameparts` -> `name_parts`